snapcraft.yaml

```diff
@@ -1359,11 +1359,9 @@ parts:
       - gdisk
       - iw
       - lshw
-      - netbase
       - pciutils
       - pigz
       - rsync
-      - squashfs-tools
```

Comment:
is the tooling still included in the snap though?

Reply:
It never made it in our snap as nothing primed it. The tools either come from `squashfs-tools` that is in the base snap or our vendored `squashfs-tools-ng`.